src/jcarbon/BUILD

```diff
@@ -21,6 +21,12 @@ java_binary(
     runtime_deps = [":jcarbon"],
 )
 
+java_binary(
```

Comment:
This rule needs to live in `src/main/java/jcarbon/emissions`. A user of our code should never be calling this.

Reply:
just made a commit to fix this problem, lmk if its good